stacks/lemp/service-config/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
```

Comment:
I do love having nginx in here.

It's semi-hilarious that we use `/var/run` when `/var/run` == `/var` for us, but I still am +1 on using `/var/run` since it looks more normal.